hadoop-plugin/src/main/groovy/com/linkedin/gradle/hadoopdsl/job/Job.groovy

```diff
@@ -329,6 +332,13 @@ class Job {
     }
   }
 
+  @HadoopDslMethod
```

Comment:
Could you add a comment here briefly describing the idea behind required parameters?